packages/app/components/NavSheet.tsx

```diff
@@ -37,7 +38,8 @@ export function NavSheet({
     >
       <Sheet.Overlay />
       <Sheet.Frame zIndex={1} height={isWeb ? '100vh' : '100%'} pt={isPwa && sat}>
-        <Stack w="100%" ai="flex-end" jc="flex-end" px="$6" pt="$6" pb="$2">
+        <XStack w="100%" ai="center" jc="space-between" px="$4" pt="$6" pb="$2">
+          <Paragraph size={'$10'}>/send</Paragraph>
```

Comment:
This doesn't match the logo styling in the rest of the app.

Let's use the svg instead
```suggestion
       <Link href="/send">
                      <IconSendLogo size={'$2.5'} color={'$color12'} />
                    </Link>
```